packages/text-serializer/test/adapter-test.js

```diff
@@ -0,0 +1,37 @@
+const { expect } = require('chai');
+const { Fury } = require('@apielements/core');
+const adapter = require('../lib/adapter');
+
+describe('Text Serializer Adapter', () => {
+  let fury;
+
+  before(() => {
+    fury = new Fury();
+    fury.use(adapter);
+  });
+
+  it('has a name', () => {
+    expect(adapter.name).to.equal('text');
+  });
+
+  it('has a text/plain media type', () => {
+    expect(adapter.mediaTypes).to.deep.equal(['text/plain']);
+  });
+
+  it('can serialize an element asynchronously', (done) => {
```

Comment:
I think it would be a good idea to have a test like this, but for the error case to count for the feedback above.

Reply:
Addressed in a131adf